README.md

```diff
@@ -3,8 +3,8 @@
 </p>
 
 <p align="center">
-  <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
+  <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+<img src="https://img.shields.io/github/contributors/sayakongit/sangnet?style=flat-square&label=All%20Contributors&color=orange" alt="contributors count" />

```

Comment:
This section is auto generated and hence it is mentioned not to change/modify this section. So it is not recommended.